docs/en/Running-Alluxio-Locally.md

```diff
@@ -34,13 +34,23 @@ repeatedly input the password, you can add the public ssh key for the host into
 $ ./bin/alluxio format
 ```
 
-# Start Alluxio Filesystem Locally
+# Step 1: Start Alluxio Filesystem Locally
 
-## Start Alluxio with sudo privileges
-
-By default, on startup Alluxio will create a
+By default, Alluxio Filesystem will create and use
 [RAMFS](https://www.kernel.org/doc/Documentation/filesystems/ramfs-rootfs-initramfs.txt) as its in-memory data storage.
-This step requires sudo privileges to perform "mount", "umount", "mkdir" and "chmod" operations. There are two approaches to achieve this:
+Depends on the operating systems and environment, it may require different privileges to complete this step.
+
+## Running on MacOS
```

Comment:
I think both MacOS and Linux require sudo, so I am not sure why are you distinguishing between the two.

What I would do instead is:

```
...

To provide memory-speed I/O, Alluxio stores data in a RAMdisk. When starting Alluxio, you can either instruct Alluxio to use an existing RAMdisk or to create the RAMdisk on your behalf (NOTE: this step requires sudo privileges).

### Starting Alluxio with Existing RAMdisk
...

### Starting Alluxio without Existing RAMdisk
...
```